### ISSUE TITLE
Shared Element Transition when viewing Images

### ISSUE THREAD

**Issue by philippschumann** [nextcloud/android#6455]
When viewing an Image in Fullscreen there is no Animation.

It would look more polished if there was a [Shared Element Transition](https://developer.android.com/training/transitions/start-activity#start-transition).

Examples of the Animation can be found at [material.io](https://material.io/develop/android/theming/motion/).

A video of a prototype I was working on is available at [youtube.com](https://youtu.be/ahd9ATT6eco).
The Animation is not perfect yet but surely can be optimized.

Let me know if you would like me to be working on this.

**Comment by tobiasKaminsky** [comment]
@jancborchardt @AndyScherzinger what do you think?
I do like it, but we then should only use it when opening images, so that there is not too much animation.

**Comment by AndyScherzinger** [comment]
> I do like it, but we then should only use it when opening images, so that there is not too much animation.

Agreed and I also like it. What needs to be checked though is how this behaves on landscape+tablet (since I am unsure if we still have the 2 column layout there).
